src/Gaffer/Reference.cpp

```diff
@@ -453,7 +438,7 @@ void Reference::loadInternal( const std::string &fileName )
 			{
 				if( newPlug->direction() == Plug::In && oldPlug->direction() == Plug::In )
 				{
-					copyInputsAndValues( oldPlug, newPlug, /* ignoreDefaultValues = */ !versionPriorTo09 );
+					copyInputsAndValues( oldPlug, newPlug, /* ignoreDefaultValues = */ true );
```

Comment:
I think it depends on how likely we are to act on this :

```/// \todo Consider moving to PlugAlgo.h```

The `ignoreDefaultValues` behaviour would be unexpected as the standard behaviour for `PlugAlgo::copyInputsAndValues()`, so I think keeping the parameter would be necessary. It seems fairly reasonable to me that this might end up being public, so would prefer to leave as-is for now.